sensor/kubernetes/listener/resources/deployment_store.go

```diff
@@ -65,7 +65,7 @@ func (ds *DeploymentStore) getDeploymentsByIDs(namespace string, idSet set.Strin
 	return deployments
 }
 
-func (ds *DeploymentStore) getMatchingDeployments(namespace string, sel selector) (matching []*deploymentWrap) {
+func (ds *DeploymentStore) getMatchingDeployments(namespace string, sw selectorWrapper) (matching []*deploymentWrap) {
```

Comment:
Right, I should change those functions using `selectorWrapper`back to using `selector`, good point. As for `labelWrapper` Piotr wants it to be `labelsWithLenImpl` so I'll go with that and I'll remember the Wrap instead of Wrapper convention for other instances like the `selectorWrapper`.